packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -369,10 +374,16 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   /// The list of native text processing actions provided by the engine.
   final List<ProcessTextAction> _processTextActions = <ProcessTextAction>[];
 
+  // Ids.
+  static int _currentSelectableId = 0;
```

Comment:
Thinking about this, I think it really depends on the app. If someone is purely relying on `SelectableRegionState.nextSelectableId` without storing them, then I think this would still work out since new ids are generated starting from 0.

If the customer is using `SelectableRegionState.nextSelectableId` and is storing these ids in their backend for re-use. Then it would be up to them to explicitly use their stored ids once they have been created. I think what is more likely in this scenario is that the customer generates unique ids on their backend and uses those from the start. Let me know if this makes sense.

Reply:
The `nextSelectableId` API has been removed.